stl/inc/xutility

```diff
@@ -21,11 +21,18 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma push_macro("new")
 #undef new
 
-#if (defined(_M_IX86) || defined(_M_X64)) && !defined(_M_CEE_PURE) && !defined(_M_HYBRID) && !defined(_M_ARM64EC)
+#if (defined(_M_IX86) || defined(_M_X64)) && !defined(_M_CEE_PURE) \
+    && !defined(_M_HYBRID) && !defined(_M_ARM64EC)

```

Comment:
I led you astray when I wrapped this line in my suggestion to keep it from being too long to fit in the comment, and now clang-format is complaining.
```suggestion
#if (defined(_M_IX86) || defined(_M_X64)) && !defined(_M_CEE_PURE) && !defined(_M_HYBRID) && !defined(_M_ARM64EC)
```